src/cli/services/linter/utils/__tests__/getResolver.test.ts

```diff
@@ -21,6 +21,6 @@ describe('getResolver', () => {
   });
 
   it('given no path, returns default resolver', () => {
-    expect(getResolver(void 0)).toBe(httpAndFileResolver);
+    expect(JSON.stringify(getResolver(void 0))).toEqual(JSON.stringify(createHttpAndFileResolver()));
```

Comment:
Mocking `createHttpAndFileResolver` would be more precise here but noticed you don't use mocks very much so wanted to keep consistent.